specification/security/resource-manager/Microsoft.Security/preview/2022-12-01-preview/defenderForStorageSettings.json

```diff
@@ -0,0 +1,250 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Security Center",
+    "description": "API spec for Microsoft.Security (Azure Security Center) resource provider",
+    "version": "2022-12-01-preview"
```

Comment:
>2022-12-01-preview [](http://example.com/codeflow?start=16&length=18)

Why use such old date? The latest date is 2023-05-01-preview:
[https://github.com/Azure/azure-rest-api-specs/tree/main/specification/security/resource-manager/Microsoft.Security/preview/2023-05-01-preview](https://github.com/Azure/azure-rest-api-specs/tree/main/specification/security/resource-manager/Microsoft.Security/preview/2023-05-01-preview)

Reply:
The API was approved before with this API version before but wasn't merged. It is already public...

Reply:
The PR that was approved was on the private repo.